packages/block-editor/src/components/link-control/style.scss

```diff
@@ -483,16 +483,11 @@ $preview-image-height: 140px;
 		 * Position spinner to the left of the actions.
 		 *
 		 * Compensate for:
-		 *  - Input margin ($grid-unit-20)
-		 *  - Border (1px)
-		 *  - Vertically, for the difference in height between the input (40px)
-		 *    and the spinner.
-		 *  - Horizontally, adjust for the width occupied by the icon buttons,
-		 *    then artificially create spacing that mimics as if the spinner
-		 *    were center-padded to the same width as an icon button.
+		 *  - Input padding top ($grid-unit-15) plus difference between effective input line-height (18px) and spinner height (16px).
+		 *  - Input padding right ($button-size)
```

Comment:
We should probably translate the calculations explained in the comment in the code too, so that the value stays correct and up to date ?

Reply:
The only issue there is that the 18px seems to be circumstantial rather than based on any registered value that I can see. 

Thinking about it a little more, we can probably just do something like `top: calc(50% - #{$spinner-size} / 2);` which seems simpler :)